xray_node/scripts/build.js

```diff
@@ -1,12 +0,0 @@
-#!/usr/bin/env node
```

Comment:
Thanks for switching to this. It was my intention to do it, but it slipped through the cracks.